src/lib/assetUtils.ts

```diff
@@ -50,3 +50,17 @@ export const getTickerFromMarketmapId = (marketmapId: string): string => {
 export const getAssetFromMarketId = (marketId: string): string => {
   return marketId.split('-')[0];
 };
+
+const SKIP_NATIVE_DENOM_SUFFIX = 'native';
+
+/**
+ * cosmjs uses this 0x address as the chain's native token.
+ * skip does not, but we add this value in order to be able to send payloads to cosmjs
+ * @todo We may need to add additional logic here if we 'useAccountBalance' on forms that do not follow this format.
+ */
+const NATIVE_TOKEN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
```

Comment:
probably better to specify that this is for cosmos native tokens? since evm native tokens dont follow the same format

```suggestion
const COSMOS_NATIVE_TOKEN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
```

Reply:
historically this format was used for non cosmos native tokens as well (at least when we were using squid). the pattern's now been standardized in our codebase. i can look into removing it but for *now* i don't think it's accurate for us to specify this as cosmos only (e.g. in the depositForm we default to the same value even though the chain we're querying is not necessarily cosmos). i think it's just the format accepted by several sdk's for native tokens

Reply:
this doesn't match up to the comment that you wrote though? it says above it's only being used for cosmjs. i also dont see this address variable used anywhere other than the deposit form right now, and it's also not the standard native token representation in skip. if there's another SDK that expects this token address format, you should prefix it with the SDK name, but it's incorrect to leave the variable name as just 'NATIVE_TOKEN_ADDRESS'

Reply:
hmm okay i can update the comment and name. i was specifically trying to call out skip since that's the only sdk that we use that specifies a different address format. we've historically used this for all SDK's dealing with token denoms/addresses until skip came along so i avoided specifics.

we can update the name to `WAGMI_COSMJS_NATIVE_TOKEN_ADDRESS` but tbh i'm leaning slightly towards `NON_SKIP_NATIVE_TOKEN_ADDRESS` since skip is the outlier here for token address formatting.

Reply:
done! actually opted for `WAGMI_COSMJS_NATIVE_TOKEN_ADDRESS` to prefer specificity for now. if the list gets unmanageably large i'm down to change it to `NON_SKIP_NATIVE_TOKEN_ADDRESS` later